shared-ui/lib/types.ts

```diff
@@ -36,3 +42,15 @@ export interface StyledFooterProps {
 export interface BackgroundProps {
   isDay: boolean;
 }
+
+export interface TeamColumnProps {
+  columnInfo: TeamColumnInfo;
+}
+
+export interface MobileTeamColumnProps {
+  listOfColumnInfo: TeamColumnInfo[];
+}
+
+export interface StyledTeamLabelProps {
+  twoLines: boolean;
+}
```

Comment:
I think this is supposed to be in the `types.ts` of the main-site directory. I would again clarify with @jzhzhang66

Reply:
from last year's i think the meet the team thingy is in both the live site and main site

Reply:
@jhylisa666 @mike10911  Oh that's a good point! Okay I'm double checking with Sarah if it's gonna be on the live-site again - will ping you once I get a response!! 

If it is going to be on the live-site, could you please move your entire meet-the-team folder into the shared-ui components folder? (and make sure all imports are updated / the site still looks good)

If it's not going to be on the live-site, could you please move the types into the main-site types file? 

Approving but please make whichever change it ends up being before you merge!

Reply:
Sounds good!!!